src/main/java/com/google/api/gax/grpc/ServiceApiSettings.java

```diff
@@ -31,6 +31,13 @@
  * and should not be used in production.
  */
 public class ServiceApiSettings {
+  // Name and version of the service generator
+  private String serviceGeneratorName;
```

Comment:
Put all of the mutable members together. (specifically, put these with the providers)